SLSA v1.0 levels - depict differences in authentication L2 vs L3 (#509)
slsa-framework/slsa: docs/spec/v1.0/levels.md

## Levels and tracks

> SLSA levels are split into *tracks*. Each track has it's own set of levels that

Reviewer: typo: its

```suggestion
SLSA levels are split into *tracks*. Each track has its own set of levels that
```

All of [Build L1], plus:

-   Builds run on a hosted build service that generates and signs the

>     provenance itself, outside the control of the users of the build service.

Author: L3 requirement: _Such secret material MUST NOT be accessible to the environment running the user-defined build steps._

> -   Build service implements strong controls to prevent runs from influencing
>     one another, even within the same project.
> 
> -   Secret material used to sign the provenance must not be accessable to the user-defined build steps.    

Reviewer: nit/suggestion: merge this with the bullet above since both are implemented by the build service. (Also fix a typo and use consistent phrasing.)

```suggestion
-   Build service implements strong controls to:
    -   prevent runs from influencing one another, even within the same project.
    -   prevent secret material used to signed the provenance from being
        accessible to the user-defined build steps.
```
